scripts/annotate/check_annotations.py

```diff
@@ -56,11 +56,19 @@ def printusage():
     df = pandas.read_csv(annoFile)
     if projectId:
         for index, row in df.iterrows():
-            csv_data.add("{}-{}".format(row["Dataset Name"],
-                                        row["Image Name"]))
+            key = "{}-{}".format(row["Dataset Name"],
+                                 row["Image Name"])
+            if key in csv_data:
+                print("Warning: Duplicate entry for image {} in dataset {}."
+                      .format(row["Image Name"], row["Dataset Name"]))
```

Comment:
As we start making progress on these tools, we will probably need to define more clearly the expectations for each scenario. In this case, two question up for discussion:

- should the tool fail-fast on the first duplicate entry?
- should the tool return a non-zero status code on any duplicate entry?

Reply:
True. Non-zero exit code would be better; still valuable to continue and print out all the duplicated entries, might be useful for fixing the annotations. And as I'm thinking about it, it should actually be an "Error" not a "Warning", as such a csv file is basically unusable, it has to be fixed.

Reply:
Makes sense to display as an ERROR rather than WARNING. I assume alongside with the error code, a follow-up action will be to use Python logging so that the level of verbosity ca be controlled with `-v[vv] / -q[qq]`